ExceptionInInitializerError / java.lang.IllegalArgumentException

from play store stats:
(244 occurrences, 12 users as of today)

## Bug description 
```
java.lang.ExceptionInInitializerError: 
  at cgeo.geocaching.storage.ContentStorage.get (ContentStorage.java:62)
  at cgeo.geocaching.utils.Log.<clinit> (Log.java:65)
  at cgeo.geocaching.utils.Log.setDebug (Log.java:102)
  at cgeo.geocaching.settings.Settings.<clinit> (Settings.java:119)
  at cgeo.geocaching.CgeoApplication.initApplicationLocale (CgeoApplication.java:116)
  at cgeo.geocaching.CgeoApplication.onCreate (CgeoApplication.java:52)
  at android.app.Instrumentation.callApplicationOnCreate (Instrumentation.java:1202)
  at android.app.ActivityThread.handleBindApplication (ActivityThread.java:7393)
  at android.app.ActivityThread.access$2400 (ActivityThread.java:251)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2284)
  at android.os.Handler.dispatchMessage (Handler.java:110)
  at android.os.Looper.loop (Looper.java:219)
  at android.app.ActivityThread.main (ActivityThread.java:8393)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:513)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1055)
Caused by: java.lang.IllegalArgumentException: 
  at android.database.DatabaseUtils.readExceptionFromParcel (DatabaseUtils.java:170)
  at android.database.DatabaseUtils.readExceptionFromParcel (DatabaseUtils.java:140)
  at android.content.ContentProviderProxy.call (ContentProviderNative.java:658)
  at android.content.ContentResolver.call (ContentResolver.java:2080)
  at android.provider.DocumentsContract.createDocument (DocumentsContract.java:1327)
  at cgeo.geocaching.storage.DocumentContentAccessor.findCreateSubdirectory (DocumentContentAccessor.java:353)
  at cgeo.geocaching.storage.DocumentContentAccessor.getSubdirUri (DocumentContentAccessor.java:323)
  at cgeo.geocaching.storage.DocumentContentAccessor.getFolderUri (DocumentContentAccessor.java:290)
  at cgeo.geocaching.storage.DocumentContentAccessor.ensureFolder (DocumentContentAccessor.java:231)
  at cgeo.geocaching.storage.ContentStorage.ensureFolder (ContentStorage.java:137)
  at cgeo.geocaching.storage.ContentStorage.ensureFolder (ContentStorage.java:127)
  at cgeo.geocaching.storage.ContentStorage.reevaluateFolderDefaults (ContentStorage.java:112)
  at cgeo.geocaching.storage.ContentStorage.<init> (ContentStorage.java:103)
  at cgeo.geocaching.storage.ContentStorage.<clinit> (ContentStorage.java:59)
```

## c:geo version 
release 2021.06.23

--- a/main/src/cgeo/geocaching/storage/ContentStorage.java
+++ b/main/src/cgeo/geocaching/storage/ContentStorage.java
@@ -101,7 +101,11 @@ public class ContentStorage {
         this.documentAccessor = new DocumentContentAccessor(this.context);
         this.fileAccessor = new FileContentAccessor(this.context);
         this.documentAccessor.refreshUriPermissionCache();
-        reevaluateFolderDefaults();
+        try {
+            reevaluateFolderDefaults();
+        } catch (RuntimeException ex) {
+            Log.e("ContentStorage: problem in initializing default folders", ex);
+        }
     }
 
     public void reevaluateFolderDefaults() {

--- a/main/src/cgeo/geocaching/storage/DocumentContentAccessor.java
+++ b/main/src/cgeo/geocaching/storage/DocumentContentAccessor.java
@@ -351,7 +351,11 @@ class DocumentContentAccessor extends AbstractContentAccessor {
             }
         }
         if (createIfNotExisting) {
-            return DocumentsContract.createDocument(getContext().getContentResolver(), dirUri, DocumentsContract.Document.MIME_TYPE_DIR, dirName);
+            try {
+                return DocumentsContract.createDocument(getContext().getContentResolver(), dirUri, DocumentsContract.Document.MIME_TYPE_DIR, dirName);
+            } catch (RuntimeException re) {
+                Log.e("Could not create dir '" + dirName + "' in '" + dirUri + "'", re);
+            }
         }
         return null;
     }

